<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>947</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Album List</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src.qrc">
    <normaloff>:/vinyl.png</normaloff>:/vinyl.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="contextMenuPolicy">
          <enum>Qt::DefaultContextMenu</enum>
         </property>
         <property name="title">
          <string>Edit</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Artist/Band</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineArtist"/>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Artist/Band from the current list</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox"/>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Album</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineAlbum"/>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Year</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineYear"/>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Rating</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="ratingSpinBox"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QPushButton" name="addButton">
              <property name="text">
               <string>Add</string>
              </property>
              <property name="icon">
               <iconset resource="src.qrc">
                <normaloff>:/add.png</normaloff>:/add.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="removeButton">
              <property name="text">
               <string>Remove selected</string>
              </property>
              <property name="icon">
               <iconset resource="src.qrc">
                <normaloff>:/remove.png</normaloff>:/remove.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="autoRepeat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>Show filters</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Sql Query</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="queryLine">
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="queryButton">
            <property name="text">
             <string>Send</string>
            </property>
            <property name="icon">
             <iconset resource="src.qrc">
              <normaloff>:/send.png</normaloff>:/send.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_6">
            <property name="title">
             <string>Filters</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <item>
              <widget class="QGroupBox" name="groupBox_4">
               <property name="title">
                <string>Show only [year]</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <widget class="QComboBox" name="onlyYearCombo"/>
                </item>
                <item>
                 <widget class="QPushButton" name="showYear">
                  <property name="text">
                   <string>Show</string>
                  </property>
                  <property name="icon">
                   <iconset resource="src.qrc">
                    <normaloff>:/show.png</normaloff>:/show.png</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_5">
               <property name="title">
                <string>Show only [rating]</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <widget class="QComboBox" name="onlyRatingCombo"/>
                </item>
                <item>
                 <widget class="QPushButton" name="showRating">
                  <property name="text">
                   <string>Show</string>
                  </property>
                  <property name="icon">
                   <iconset resource="src.qrc">
                    <normaloff>:/show.png</normaloff>:/show.png</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_3">
               <property name="title">
                <string>Show only [artist]</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QComboBox" name="onlyArtistCombo"/>
                </item>
                <item>
                 <widget class="QPushButton" name="showArtistsButton">
                  <property name="text">
                   <string>Show</string>
                  </property>
                  <property name="icon">
                   <iconset resource="src.qrc">
                    <normaloff>:/show.png</normaloff>:/show.png</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
  <zorder>tableView</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox_2</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
